corehq/apps/es/management/commands/elastic_sync_multiplexed.py

```diff
@@ -79,6 +80,16 @@ def _prepare_index_for_reindex(self, index_name):
         es_manager.index_configure_for_reindex(index_name)
         es_manager.index_set_replicas(index_name, 0)
 
+    def _wait_for_index_to_get_healthy(self, index_name, sleep_time=0):
+        for i in range(10):
+            health = es_manager.cluster_health(index=index_name)
+            status = health["status"]
+            if status == "green":
+                break
+
+            print(f"\tWaiting for index status to be green. Current status: '{status}'")
+            time.sleep(sleep_time)
```

Comment:
Looks like `sleep_time` defaults to 0, which means this could happen really fast. Maybe use exponential back-off with a max wait?
```suggestion
            time.sleep(max(sleep_time, min(2 ** i, 30)))
```

Reply:
I am passing in sleep time of 5 when I was calling it. https://github.com/dimagi/commcare-hq/pull/33165/commits/a132cb73a0b64dd2e428c344af91fda46a06f155#diff-bf5c972d6375fbf6e994cc17b64951745ec9487f9071be21264eb7d9186a3326R101

But I like the idea of exponential wait off.